pkg/lifecycle/render/config/daemonresolver_test.go

```diff
@@ -147,19 +148,24 @@ func TestDaemonResolver(t *testing.T) {
 			}
 
 			daemonCtx, daemonCancelFunc := context.WithCancel(context.Background())
-			defer daemonCancelFunc()
+			daemonCloseChan := make(chan struct{})
 
-			log.Log("starting daemon")
-			go func() {
+			require.NoError(t, log.Log("starting daemon"))
```

Comment:
Still not super psyched about checking errors from go-kit/log but you're fixing this race-y test so you do you